### PR TITLE
Replace modifiers and blueprint config with one ontology_config.json

### PR DESCRIPTION
**Pull request by ewelinagr**
Related tickets:
[TMT-974](https://jira.thehyve.nl/browse/TMT-974)
[TMT-975](https://jira.thehyve.nl/browse/TMT-975)

TODO:
- extend validation of ontology_config.json to check if a concept is a part of CSR model (pydantic validator?). Now it would fail, but the validation could be more explicit, maybe also extended with other validations.
- fix issue with pydantic not mapping correctly self-referenced tree node children

### CODE REVIEW

**Comment by gijskant** [comment]
Don't forget to add `csr2transmart.mappers` to `packages` in `setup.py`

File: csr2transmart/mappers/ontology_mapper.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        entity_types = list(SubjectEntity.__args__)
+        entity_types.extend(list(StudyEntity.__args__))
+        entity_type: SubjectEntity = list(filter(lambda x: x.schema()['title'] == entity_name, entity_types))[0]
+        field_type = entity_type.schema()['properties'][entity_field_name]['type']
+        return self.type_to_value_type(field_type)
+
+    def map_concept_node(self, node: OntologyConfigConceptNode, path_elements: List[str]) -> ConceptNode:
+        entity_name, entity_field_name = node.concept_code.split('.')
+        path_elements.append(entity_field_name)
+        concept_path = '\\'.join(path_elements)
```
Comment (gijskant) [inline csr2transmart/mappers/ontology_mapper.py]: This should perhaps be `'\\\\'` then, looking at what ends up in the generated files

Comment (gijskant) [reply csr2transmart/mappers/ontology_mapper.py]: The `path_elements` list seems to be growing? E.g, `\Central Subject Registry\\01. Patient information\gender\birth_date\taxonomy\death_date\Informed_consent\ic_type\ic_version`...
We can also choose to have the concept path `'\\\\CSR\\\\' + node.concept_code`.  It does not have to be the same as the tree node path.

Comment (ewelinagr) [reply csr2transmart/mappers/ontology_mapper.py]: Good idea, I changed this to the simplified version: `'\\\\CSR\\\\' + node.concept_code`

**Pull request merged**